staging/src/k8s.io/apiserver/pkg/server/filters/timeout.go

```diff
@@ -93,6 +93,10 @@ func (t *timeoutHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	resultCh := make(chan interface{})
 	var tw timeoutWriter
 	tw, w = newTimeoutWriter(w)
+
+	//Make a copy of request and work on it in new goroutine
```

Comment:
Please add a space to all comments between "//" and text to be consistent with other comments in this repo.